modules/utility/init.zsh

```diff
@@ -175,9 +175,13 @@ elif (( $+commands[wget] )); then
   alias get='wget --continue --progress=bar --timestamping'
 fi
 
-# Resource Usage
-if (( $+commands[pydf] )); then
-  alias df=pydf
+if zstyle -s ':prezto:module:utility' pydf 'yes'; then
```

Comment:
I'm tempted to take the sledgehammer approach and just say remove pydf in general. We killed branches in `diff` which fell back to commands which didn't support the same options... maybe it would be worth doing the same here.